dropwizard-core/src/main/java/io/dropwizard/server/AdminServletFactory.java

```diff
@@ -0,0 +1,294 @@
+package io.dropwizard.server;
+
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.health.HealthCheckRegistry;
+import com.codahale.metrics.servlets.AdminServlet;
+import com.codahale.metrics.servlets.HealthCheckServlet;
+import com.codahale.metrics.servlets.MetricsServlet;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.dropwizard.jetty.MutableServletContextHandler;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Can we avoid bringing commons-lang3 back in?

Reply:
Since only `StringUtils.isNotEmpty` is used, I think this can simply be replaced by a small helper method.